src/hooks/useProtocolMetrics.ts

```diff
@@ -123,15 +132,69 @@ export const useProtocolMetrics = <TSelectData = unknown>(select?: (data: Protoc
   );
 };
 
-export const useMarketCap = () => useProtocolMetrics(metrics => metrics[0].marketCap);
-export const useTotalSupply = () => useProtocolMetrics(metrics => metrics[0].totalSupply);
-export const useTotalValueDeposited = () => useProtocolMetrics(metrics => metrics[0].totalValueLocked);
-export const useTreasuryMarketValue = () => useProtocolMetrics(metrics => metrics[0].treasuryMarketValue);
-export const useTreasuryLiquidBacking = () => useProtocolMetrics(metrics => metrics[0].treasuryLiquidBacking);
+export const useMarketCap = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].marketCap },
+  );
+export const useTotalSupply = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].totalSupply },
+  );
+export const useTotalValueDeposited = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].totalValueLocked },
+  );
+export const useTreasuryMarketValue = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].treasuryMarketValue },
+  );
 export const useTreasuryLiquidBackingPerOhmFloating = () =>
-  useProtocolMetrics(metrics => metrics[0].treasuryLiquidBackingPerOhmFloating);
-export const useOhmCirculatingSupply = () => useProtocolMetrics(metrics => metrics[0].ohmCirculatingSupply);
-export const useOhmFloatingSupply = () => useProtocolMetrics(metrics => metrics[0].ohmFloatingSupply);
-export const useOhmPrice = () => useProtocolMetrics(metrics => metrics[0].ohmPrice);
-export const useGOhmPrice = () => useProtocolMetrics(metrics => metrics[0].gOhmPrice);
-export const useCurrentIndex = () => useProtocolMetrics(metrics => metrics[0].currentIndex);
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].treasuryLiquidBackingPerOhmFloating },
+  );
+export const useOhmCirculatingSupply = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].ohmCirculatingSupply },
+  );
+export const useOhmFloatingSupply = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].ohmFloatingSupply },
+  );
+export const useOhmPrice = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].ohmPrice },
+  );
+export const useGOhmPrice = () =>
+  useMetricsBarLatestOnlyQuery(
+    { endpoint: getSubgraphUrl() },
+    {},
+    { select: data => data.protocolMetrics[0].gOhmPrice },
+  );
```

Comment:
can you add a dev note like we used here: https://github.com/OlympusDAO/olympus-frontend/pull/2042/files#diff-af3068af5f21a4472ce4cdc8d986e9604ea507eed720efd2e5d8d67c46578460R137

Maybe it should be more specific than what I wrote about how it should only be used on the dashboard.